spec/MultiDomainInventoryProxy.yaml

```diff
@@ -12344,6 +12358,551 @@ paths:
+                from [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-op-s-bm-014/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-configuration/life-cycle-state]'
+        '400':
+          $ref: '#/components/responses/responseForErroredServiceRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredServiceRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredServiceRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredServiceRequests'
+  /v1/update-client-of-subsequent-release:
+    parameters:
+      - $ref: '#/components/parameters/user'
+      - $ref: '#/components/parameters/originator'
+      - $ref: '#/components/parameters/x-correlator'
+      - $ref: '#/components/parameters/trace-indicator'
+      - $ref: '#/components/parameters/customer-journey'
+    post:
+      operationId: UpdateClientOfSubsequentRelease
+      summary: 'Configures Http and TcpClient of the NewRelease'
+      tags:
+        - BasicServices
+      security:
+        - apiKeyAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - application-name
+                - release-number
+                - protocol
+                - address
+                - port
+              properties:
+                application-name:
+                  type: string
+                  description: >
+                    'Name of the application that is target of the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-1-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
+                release-number:
+                  type: string
+                  pattern: '^([0-9]{1,2})\.([0-9]{1,2})\.([0-9]{1,2})$'
+                  description: >
+                    'Release of the application that is target of the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-1-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/release-number]'
+                protocol:
+                  type: string
+                  enum:
+                    - 'HTTP'
+                    - 'HTTPS'
+                  description: >
+                    'Protocol to be used during the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-tcp-c-ro-1-0-2-000/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-protocol]'
```

Comment:
mdip-1-0-2-tcp-c-ro-1-0-2-000 should be changed to mdip-1-0-2-tcp-c-nr-1-0-2-000. Also to the port and protocol.